src/custom/pages/LimitOrder/pureComponents/ReceiveAmountInfo/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
Not used

Reply:
Fixed: https://github.com/cowprotocol/cowswap/pull/1049/commits/58fb25dc9a70893e4becb49ce8778592cac53eac